reactioncommerce/reaction
# (refactor): remove meteor-collection-hooks dependency for orders

This PR removes the meteor-collection-hooks package dependency for new orders. `Hooks.Events` is now used to achieve the same functionality 

**Testing Procedure**
- Place an order by completing the checkout process
- Verify that a new AnalyticsEvent document with a field named value is set to the new order's `_id`
